src/OpenTK.Windowing.Common/VSyncMode.cs

```diff
@@ -1,4 +1,4 @@
-namespace OpenToolkit.Windowing.Common
+﻿namespace OpenToolkit.Windowing
```

Comment:
Why was this namespace changed?

Reply:
good question, perhaps happened cause of rebasing. gonna change it back